### ISSUE TITLE
Movie posters not showing for requests

### ISSUE THREAD

**Issue by ScouseTS** [tidusjar/PlexRequests.Net#675]
If this is a bug report please make sure you have filled the following in:
(If it's not a bug and a feature request then just remove the below)

#### Plex Requests.Net Version:
1.9.7

#### Operating System:
Server 2012

#### Mono Version:
Unsure

#### Applicable Logs (from `/logs/` directory or the Admin page):

```
Logs go here (Please make sure you remove any personal information from the logs)
```

#### Reproduction Steps:
When attempting to view any movie requests that have been made no posters are showing, however for TV Shows the posters are showing fine

Please include any steps to reproduce the issue, this the request that is causing the problem etc.

It was happening on previous version 1.9.0, not sure why it just stopped working, would appreciate any help.

Thanks

**Comment by tidusjar** [comment]
Can you open the browsers console and use the element inspector (top left in chrome, the box with the arrow) and "inspect" the image, can you post the results then?

**Comment by ScouseTS** [comment]
`Failed to load resource: the server responded with a status of 404 ()
https://image.tmdb.org/t/p/w150/https://image.tmdb.org/t/p/w150//5vwP5vUBqizNI5O7UU6mAgkTkGa.jpg Failed to load resource: the server responded with a status of 404 ()
https://image.tmdb.org/t/p/w150/https://image.tmdb.org/t/p/w150//tDFNUuYkQ4ZeAlrBHeTtimAumXo.jpg Failed to load resource: the server responded with a status of 404 ()
https://image.tmdb.org/t/p/w150/https://image.tmdb.org/t/p/w150//xfMQlAiM3OnhSnkyNDQBUfokbKy.jpg Failed to load resource: the server responded with a status of 404 ()
https://image.tmdb.org/t/p/w150/https://image.tmdb.org/t/p/w150//aqhAqttDq7zgsTaBHtCD8wmTk6k.jpg Failed to load resource: the server responded with a status of 404 ()`

:)

**Comment by tidusjar** [comment]
Thank you, is this with new requests or old ones? (Before version 1.9.7)

**Comment by ScouseTS** [comment]
Both, all requested listed have no posters but I know for older requests they did exist :)

**Comment by tidusjar** [comment]
What new requests did this happen for? (So i can reproduce this issue)

**Comment by ScouseTS** [comment]
Any movies one, I think when searching for the movie the posters appear but when I go into admin to see any requests made that is when they don't link up correctly.

**Comment by SuperPotatoMen** [comment]
@ScouseTS  Pleas give us some movie names. I was trying for reproduce but i can't find the problem.
If you on 1.9.7 we save the movie posters not with the image link: "https://image.tmdb.org/t/p/w150/https://image.tmdb.org/t/p/w150//aqhAqttDq7zgsTaBHtCD8wmTk6k.jpg " but only like "https://image.tmdb.org/t/p/w150//aqhAqttDq7zgsTaBHtCD8wmTk6k.jpg"

**Comment by ScouseTS** [comment]
Status
Version:  1.9.7
Update Available:  X

Micky Flanagan: Live - The Out Out Tour (2011)
Bad Santa (2003)

My plexrequest is public facing... happy to provide creds so you can login and check yourself :)

**Comment by SuperPotatoMen** [comment]
@ScouseTS  is it possible to have acces on youre settup?

**Comment by ScouseTS** [comment]
@SuperPotatoMen Please send me your email address... can't figure out how to PM on here.

**Comment by SuperPotatoMen** [comment]
@ScouseTS can you go on the chat app on the frontpage?  You can om there

**Comment by my95z34** [comment]
I've been having this issue as well.  Seen here: http://imgur.com/a/NVSTx

Edge of Seventeen shows this URL : https://image.tmdb.org/t/p/w150/https://image.tmdb.org/t/p/w150//gxK80WUwkQePpXpzRdpob0i7DD5.jpg

Live by Night : https://image.tmdb.org/t/p/w150/https://image.tmdb.org/t/p/w150//utGq0tOcAPjm5DNMMAbDZi0JTik.jpg

And Ice Age : https://image.tmdb.org/t/p/w150/https://image.tmdb.org/t/p/w150//dN3KkgXcDkP1QkOZPz5EGXzS8B7.jpg

If you need any more info, lemme know!

**Comment by SuperPotatoMen** [comment]
Are the movies added after upgraded to version 1.9.7?

**Comment by SuperPotatoMen** [comment]
@tidusjar is the fix for the poster art included for the movies or only for the tv shows?

**Comment by my95z34** [comment]
I'm on 1.9.7 in my screenshots I posted.

**Comment by SuperPotatoMen** [comment]
This problem make no sence because it is fixed.  I'm confused

**Comment by my95z34** [comment]
Does it fix old requests or just ones made after the update to  1.7.9?

**Comment by SuperPotatoMen** [comment]
No, it Fixed only for New Request after a 1.9.7 update.  Old Requests doesn't still have the wrong iris in the database.  Thats why I'm ask If the Requests are made before you update too the 1.9.7 version

**Comment by my95z34** [comment]
Shoot, sorry.  Didn't realize that.  The requests I'm looking at are from
prior to 1.9.7.  All new requests seem to work correctly.  Is there any way
to apply the fix to old requests?

**Comment by SuperPotatoMen** [comment]
https://github.com/tidusjar/PlexRequests.Net/issues/657 it is fixed already.  You can wait for the beta release.  It will be out soon between now and two weeks.

**Comment by SuperPotatoMen** [comment]
Fix will be live in the 1.10.0.  pleas wait for the new release.
